Packs/MitreCaldera/Integrations/MitreCaldera/MitreCaldera.py

```diff
@@ -0,0 +1,1885 @@
+    )
+    return command_results
+
+
+def delete_adversary_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    adversary_id = args.get('adversary_id')
+
+    client.delete_adversary(adversary_id)
+    command_results = CommandResults(
+        readable_output=f"Adversary with ID {adversary_id} was deleted successfully."
+    )
+    return command_results
+
+
+def delete_schedule_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    schedule_id = args.get('schedule_id')
+
+    client.delete_schedule(schedule_id)
+    
+    command_results = CommandResults(
+        readable_outputs=f"Schedule with ID {schedule_id} deleted successfully."
+    )
+    return command_results
+
+
+def get_abilities_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    ability_id = args.get('ability_id')
+    sort = args.get('sort')
+    include = argToList(args.get('include', []))
+    exclude = argToList(args.get('exclude', []))
+    if ability_id:
+        response = client.get_abilities_by_ability_id(ability_id, include, exclude)
+    else:
+        response = client.get_abilities(sort, include, exclude)
+    command_results = CommandResults(
+        outputs_prefix='MitreCaldera.Abilities',
+        outputs_key_field='ability_id',
+        outputs=response,
+        raw_response=response
+    )
+
+    return command_results
+
+
+def get_adversaries_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    adversary_id = args.get('adversary_id')
+    sort = args.get('sort')
+    include = argToList(args.get('include', []))
+    exclude = argToList(args.get('exclude', []))
+
+    if adversary_id:
+        response = client.get_adversaries_by_adversary_id(adversary_id, include, exclude)
+    else:
+        response = client.get_adversaries(sort, include, exclude)
+    command_results = CommandResults(
+        outputs_prefix='MitreCaldera.Adversaries',
+        outputs_key_field='adversary_id',
+        outputs=response,
+        raw_response=response
+    )
+
+    return command_results
+
+
+def get_agents_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    paw = args.get('paw')
+    sort = args.get('sort')
+    include = argToList(args.get('include', []))
+    exclude = argToList(args.get('exclude', []))
+
+    if paw:
+        response = client.get_agents_by_paw(paw, include, exclude)
+    else:
+        response = client.get_agents_request(sort, include, exclude)
+    command_results = CommandResults(
+        outputs_prefix='MitreCaldera.Agents',
+        outputs_key_field='',
+        outputs=response,
+        raw_response=response
+    )
+
+    return command_results
+
+
+def get_config_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    name = args.get('name')
+
+    response = client.get_config_by_name(name)
+    response['name'] = name
+    command_results = CommandResults(
+        outputs_prefix='MitreCaldera.Config',
+        outputs_key_field='name',
```

Comment:
Thanks for your fix in the yml file 
but still, the key defined in the yml are now `MitreCaldera.Name`
while from the code - the key will be under `MitreCaldera.Config.name`